docs/contributing.md

```diff
@@ -66,18 +66,20 @@ Install the following dependencies before developing on conda-store.
 Install the development dependencies and activate the environment.
 
 ```shell
+# replace this with environment-macos-dev.yaml or environment-windows-dev.yaml
+# if you are on Mac or Windows
 conda env create -f conda-store-server/environment-dev.yaml
 conda activate conda-store-server-dev
 ```
 
 Running `conda-store`. `--standalone` mode launched celery as a
-subprocess of the web server.
+subprocess of the web server. Run
 
-python -m conda_store_server.server --standalone tests/assets/conda_store_standalone_config.py
-
-````
+```
+python -m conda_store_server.server --standalone
+```
```

Comment:
The old command was wrong because it didn't pass the config option. But why does the new example not tell how to run with a config? See https://github.com/conda-incubator/conda-store/issues/594

Reply:
The idea behind --standalone is that the config defaults are suitable for standalone mode. So most developers shouldn't need to pass a config, unless they specifically need to change a given config value.

Reply:
OK, the option is available in help anyway. Let's keep it simple.